lte/gateway/c/core/oai/tasks/nas5g/src/M5GPDUSessionEstablishmentAccept.cpp

```diff
@@ -27,8 +27,7 @@ PDUSessionEstablishmentAcceptMsg::~PDUSessionEstablishmentAcceptMsg(){};
 
 // Decode PDUSessionEstablishmentAccept Message and its IEs
 int PDUSessionEstablishmentAcceptMsg::DecodePDUSessionEstablishmentAcceptMsg(
-    PDUSessionEstablishmentAcceptMsg* pdu_session_estab_accept, uint8_t* buffer,
-    uint32_t len) {
+    uint8_t* buffer, uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.